### PR TITLE
Add A Few Unit Tests for Automation Script

### PR DESCRIPTION
**Pull request by jayjayjpg**
This adds a couple of tests for the goodbits templating script. This also extracts several of the functions used in the script to make it easier to test separate parts of the workflow.

You can run the test suite via `npm run test`. This will boot up a mock page of an example blog post required for the tests and will subsequently run the test modules found in `/tests`

This depends on https://github.com/ember-learn/ember-blog/pull/149

## TODOs:
- [ ] setup travis
- [ ] add test step to travis workflow

### CODE REVIEW

**Comment by jayjayjpg** [comment]
Can you take a look when you find time as well @tylerturdenpants or @amyrlam? ✨

**Review by tylerturdenpants — COMMENTED** [review]
I would like to fix the indentation to 2 spaces! I have see. The error I my ways using tabs 😉

File: lib/get-content.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+			if (paragraphs) {
+				return [...paragraphs].map((para) => {
+					return para.outerHTML;
+				}).join("");
+			}
+			return "";
+		}
+
+		getParagraphs = (index) => {
+			return document.querySelectorAll(`[data-embertimes-section-fragment="${index}"]`);
```
Comment (jayjayjpg) [inline lib/get-content.js]: This should allow us to fetch all the text elements for a particular section, once https://github.com/ember-learn/ember-blog/pull/149 has landed

**Comment by amyrlam** [comment]
Let's merge this in, filed an issue for the todos https://github.com/ember-learn/ember-times-tools/issues/29, worked like a charm last week.

@tylerturdenpants You want linting added here?

**Pull request merged**